dangerfile.ts

```diff
@@ -4,12 +4,11 @@ import { compact, includes, remove } from "lodash"
 
 // For now you can ignore these 3 errors, I'm not sure why
 // they are being raised. They definitely exist in the node runtime.
-import * as fs from "fs"
-import * as os from "os"
-import * as path from "path"
+const fs = require("fs") // tslint:disable-line
+const os = require("os") // tslint:disable-line
+const path = require("path") // tslint:disable-line
```

Comment:
no idea why, but in _this_ project those imports give tsc errors as `import`s works fine in Danger and Peril.

Converting them to `require`s allows tsserver / tslint to continue evaluating.